configure.py

```diff
@@ -2315,6 +2315,9 @@ def test_exe_extra_ldflags():
         if osinfo.soname_pattern_patch is not None:
             variables['soname_patch'] = osinfo.soname_pattern_patch.format(**variables)
 
+        if options.os == 'windows':
+            variables['implib_name'] = variables['static_lib_name']
+
```

Comment:
On Windows, the CMake package config must be explicitly aware of the "implib" (i.e. the `.lib` file accompanying the `.dll`). Before, we used `static_lib_name` implicitly where needed (mostly in the `install.py`). I cleaned that up a little, @randombit, just so you are aware.

Reply:
:+1: